webapp/app.py

```diff
@@ -146,7 +147,24 @@ def global_template_context():
     version_parts = VANILLA_VERSION.split(".")
     version_minor = f"{version_parts[0]}.{version_parts[1]}"
 
-    return {"version": VANILLA_VERSION, "versionMinor": version_minor, "path": flask.request.path}
+    docs_slug = flask.request.path.replace("/docs/", "")
+    docs_slug = docs_slug.replace("/design/", "")
```

Comment:
I'm sure there's a good reason not to... but why not chaining the replace method instead of reassigning the variable?

Reply:
The max 80 char per line rule.

Reply:
I think something like this would do the trick: (It's not particularly pretty, though... Feel free to leave as it is)
```py
docs_slug = (
    flask.request.path.replace("/docs/", "")
    .replace("/design/", "")
    .replace("/accessibility", "")
)
```

Reply:
Good suggestion! I will try it.